lib/menus/components/common.js

```diff
@@ -42,6 +42,12 @@ const commonCss = css`
   }
 `
 
+/** @returns {boolean} */
+function isMobile () {
+  const check = false
```

Comment:
Looks like this at least need fixing

Reply:
I search for many possible solutions but there are people who think the userAgent is not reliable. Id really know what the best solution is. Maybe we could detect if it is a mobile device if the person uses touch or mouse when clicking on the play button.

Reply:
or maybe use this lib "ismobilejs". (pixijs uses it)